external/java8-tests/src/test/java/test/org/apache/spark/java8/Java8RDDAPISuite.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.spark;
+package test.org.apache.spark.java8;
```

Comment:
Yeah, core tests use `test.org.apache.spark` to specifically test from outside the same package. Is `.java8` needed? I don't particularly care much myself. Some scala tests use `org.apache.sparktest`, unfortunately.